src/main/frontend/common/i18n/messages.ts

```diff
@@ -63,16 +66,34 @@ export async function getMessages(
   return new Messages(messages, locale);
 }
 
+export type MessageKeyType = LocalizedMessageKey | string;
+
+export enum LocalizedMessageKey {
+  millisecond = "Util.millisecond",
+  second = "Util.second",
+  minute = "Util.minute",
+  hour = "Util.hour",
+  day = "Util.day",
+  month = "Util.month",
+  year = "Util.year",
```

Comment:
do we want to use this as an opportunity to remove the util prefix? maybe replace it with `timings` or something like or just nothing

Reply:
definitely don't need the util prefix, (may need to manually delete and re-import in crowdin)